### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ik-web**

### CODE REVIEW

File: src/transformState.js [hunk 1]
```diff
@@ -5,7 +5,23 @@
...
+      for (const el of obj.keysToRemove) {
+        if (state.hasOwnProperty(el)) {
+          delete state[el];
+        }
+      }
+    } else if (obj.type === 'clear') {
+      for (const deletedItem in state) {
+        delete state[deletedItem];
+      }
+    }
```
Comment (illiarerun) [inline src/transformState.js:21]: Looks complicated, that's a great place to use switch/case

File: src/transformState.js [hunk 2]
```diff
@@ -5,7 +5,23 @@
...
+  }
+
+  return state;
```
Comment (illiarerun) [inline src/transformState.js:24]: it's a void f no need to return anything here

File: src/transformState.js (hunk 1 above)
Comment (ik-web) [reply src/transformState.js]: Fixed

**Review by illiarerun — APPROVED** [review]
GJ

**Pull request closed without merging**